### PR TITLE
feat: support host flag in more upload and activate commands

### PR DESCRIPTION
**Pull request by ivo-toby**

### CODE REVIEW

File: packages/contentful--create-contentful-app/package-lock.json
Comment (andipaetzold) [inline packages/contentful--create-contentful-app/package-lock.json:1]: Nothing should change here

File: packages/contentful--app-scripts/bin/app-scripts [hunk 1]
```diff
@@ -29,6 +29,7 @@ async function runCommand(command, options) {
     .option('--token [accessToken]', 'Your content management access token')
     .option('--comment [comment]', 'Optional comment for the created bundle')
     .option('--skip-activation', 'A Boolean flag to skip automatic activation')
+    .option('--host [host]', 'Contentful CMA-endpoint to use')
```
Comment (andipaetzold) [inline packages/contentful--app-scripts/bin/app-scripts:32]: We don't specify the endpoint here. We only specify the domain/host.

Also, imo the word "CMA-endpoint" doesn't look pretty 😬

File: packages/contentful--app-scripts/lib/activate/activate-bundle.test.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ const mockedSettings = {
 };
 
 describe('activate-bundle', () => {
-  let activateBundle, clientMock, updateStub;
+  // eslint-disable-next-line no-unused-vars
```
Comment (andipaetzold) [inline packages/contentful--app-scripts/lib/activate/activate-bundle.test.js:13]: Why unused?

**Review by andipaetzold — APPROVED** [review]
just nits

File: packages/contentful--app-scripts/lib/activate/activate-bundle.test.js (hunk 2 above)
Comment (ivo-toby) [reply packages/contentful--app-scripts/lib/activate/activate-bundle.test.js:13]: Code was nagging about it, though it's actually used

**Pull request merged**